python/pyspark/rdd.py

```diff
@@ -173,6 +173,7 @@ def ignore_unicode_prefix(f):
     return f
 
 
+
```

Comment:
btw, I would revert unrelated changes to make the backport easier.

Reply:
I am not really familiar with the codebase, can you provide more details please?

Reply:
this new line is an unnecessary change